scripts/install-managed-daemons.sh

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env bash
+set -ex
+
+if [ -n "$AIR_GAPPED" ]; then
+    echo "Air-gapped region, assuming ecs-init and dependencies will be in additional-packages/ directory"
+    exit 0
+fi
+
+WORK_DIR="$(mktemp -d)"
+trap "rm -rf ${WORK_DIR}" EXIT
+
+# gpg key is taken from https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-agent-install.html
+cat >"$WORK_DIR/amazon-ecs-agent.gpg" <<EOF
```

Comment:
We have this key in https://github.com/aws/amazon-ecs-agent/blob/master/scripts/amazon-ecs-agent.gpg too, do we want to download from there instead of keeping another copy? No strong opinion though.

Reply:
I believe we used to this, but stopped because cross-partition downloads can be flaky and sometimes extremely slow